hassio/src/ingress-view/hassio-ingress-view.ts

```diff
@@ -44,18 +47,106 @@ class HassioIngressView extends LitElement {
 
   private _sessionKeepAlive?: number;
 
+  private _resolveIngressURL: {
+    status?: number;
+    time?: number;
+    interval?: number;
+  } = {};
+
   public disconnectedCallback() {
     super.disconnectedCallback();
 
     if (this._sessionKeepAlive) {
       clearInterval(this._sessionKeepAlive);
       this._sessionKeepAlive = undefined;
     }
+    if (this._resolveIngressURL.interval) {
+      clearInterval(this._resolveIngressURL.interval);
+    }
+    this._resolveIngressURL = {};
   }
 
-  protected render(): TemplateResult {
+  public connectedCallback() {
+    super.connectedCallback();
+
+    this._resolveURL();
+    this._resolveIngressURL.interval = window.setInterval(async () => {
+      await this._resolveURL();
+    }, 1000);
```

Comment:
Wouldnt it be better to let `this._resolveURL` schedule a new check if it wasn't successful?

Reply:
That works too, but better 🤷‍♂️ 
It will be less code